integrations/access/common/app.go

```diff
@@ -390,7 +390,12 @@ func (a *BaseApp) getMessageRecipients(ctx context.Context, req types.AccessRequ
 	recipientSet := NewRecipientSet()
 
 	switch a.Conf.GetPluginType() {
-	case types.PluginTypeOpsgenie, types.PluginTypeServiceNow:
+	case types.PluginTypeServiceNow:
+		// The ServiceNow plugin does not use recipients currently and create incidents in the incident table directly.
+		// Recipients just needs to be non empty.
+		recipientSet.Add(Recipient{})
```

Comment:
This is a little confusing, why adding an empty recipient, can't we return an empty list here?

Reply:
The way the common.app is setup it assumes all plugins have a concept of a recipient, e.g channels for the slack one. But since all servicenow incidents go straight to the servicenow incident table it doesn't need one. The empty is just so when app.onPendingRequest calls getRecipients and loops over them there is something in the slice when servicenow is th eplugin being used.

Would replacing the empty Recipient with one where the recipient kind is 'servicenow- instance' be clearer?
Then just always use that as the sole recipient  when using the servicenow plugin

Reply:
If the current method is fine i can go ahead and merge this

Reply:
Might be worth while making some general way for plugins to mark whether or not they have recipients as a concep